batch/sql/add_azure_tables.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+import os
+
+from gear import Database
+
+
+async def main():
+    db = Database()
+    try:
+        await db.async_init()
+
+        cloud = os.environ['HAIL_CLOUD']
+        if cloud != 'azure':
+            return
+
+        scope = os.environ['HAIL_SCOPE']
+
+        await db.just_execute('''
+DELETE FROM `pools`;
+''')
+        await db.just_execute('''
+DELETE FROM `inst_colls`;
+''')
+
+        enable_standing_worker = scope != 'dev'
```

Comment:
This needs to be removed here and used down below where the pools are defined. The `enable_standing_worker` flag should only be used for the 'standard' pool.